src/components/CustomDisciplineMenuItem.tsx

```diff
@@ -44,24 +37,10 @@ const ErrorText = styled.Text`
   margin: 10px;
 `
 
-const DeleteContainer = styled.View`
-  width: ${widthOfTrashButton};
-`
-
-const DeleteButton = styled(RectButton)`
-  height: ${wp('21%')}px;
-  margin: 0 16px 8px -16px;
-  align-items: center;
-  flex: 1;
-  justify-content: center;
-  background-color: ${props => props.theme.colors.lunesFunctionalIncorrectDark};
-  width: ${widthOfTrashButton};
-`
-
 interface CustomDisciplineMenuItemPropsType {
   apiKey: string
   selectedId: string | null
-  setSelectedId: (input: string) => void
+  setSelectedId: React.Dispatch<React.SetStateAction<string | null>>
```

Comment:
Why do we need this type here? Just `(selectedId: number) => void` should be enough.